input/pages/scopes-and-launch-context.md

```diff
@@ -35,8 +35,8 @@ permissions.
 
 For example:
 
-* If a client uses SMART App Launch to request `user/*.cruds` and is granted these scopes by a user, these scopes convey "full access" relative to the user's underlying permissions.  If the underlying user has limited permissions, the client will face these same limitations.
-* If a client uses SMART Backend Services to request `system/*.cruds`, these scopes convey "full access" relative to a pre-configured client policy.  If the pre-configured policy imposes limited permissions, the client will face these same limitations.
+* If a client uses SMART App Launch to request `user/*.cruds` and is granted these scopes by a user, these scopes convey "full access" relative to the server's underlying permissions.  If the underlying user has limited permissions, the client will face these same limitations.
```

Comment:
```suggestion
* If a client uses SMART App Launch to request `user/*.cruds` and is granted these scopes by a user, these scopes convey "full access" relative to the user's underlying permissions.  If the underlying user has limited permissions, the client will face these same limitations.
```

The *user*'s permissions are being delegated. This language achieve consensus through pretty careful dissection; if we want to add to it, I'd suggest bringing a proposal back to FHIR-I.

Reply:
OK I will create a tracker,  I am not sure who the user is or what his role is.